war/src/main/js/templates/firstUserPanel.hbs

```diff
@@ -3,7 +3,7 @@
 </div>
 <div class="modal-body">
 	<div class="jumbotron welcome-panel security-panel">
-		<iframe src="{{baseUrl}}/setupWizard/setupWizardFirstUser"></iframe>
+		<iframe src="{{baseUrl}}/setupWizard/setupWizardFirstUser" class="setup-first-user"></iframe>
```

Comment:
Where is define the CSS class `setup-first-user`?

Reply:
If you are using a CSS class as selector, please, use `id` attribute.

Reply:
👎 _always_ use CSS selectors.

Reply:
@kzantow I'm sorry but I disagree, specially, in that case where you are 100% about the `id` for this `iframe`. CSS Classes without semantic only for selecting... 🐼

Reply:
@recena It does have some meaning, it indicates that this is the `iframe` where the first user is configured. It's just not used for styling. I wasn't using an id because I'm conditioned to avoid it, plus as I'm dynamically adding it in JS, I don't consider it to be really unique any more. Granted, there normally won't be multiple of this at the same time, but it still doesn't really identify a single element imo.

Reply:
Although in this case, it *is* used to identify a single element, so I'll change it now, I can change it back if the discussion results in something else

Reply:
@xxyy My personal _rule_ is if the element is 100% unique and I'm totally sure, use `id`. In that case, it seems the `iframe` will be unique.